edx_sga/tests.py

```diff
@@ -207,6 +211,10 @@ def test_student_view_with_score(self, Fragment, render_template):
         self.assertEqual(context['is_course_staff'], True)
         self.assertEqual(context['id'], 'name')
         student_state = json.loads(context['student_state'])
+        self.assertEqual(
+            student_state['display_name'],
+            "Staff Graded Assignment"
+        )
```

Comment:
Both of these tests assert the display name is the default one, we should have a test that asserts it can be changed.